types/test/test_chronology.c

```diff
@@ -8,16 +8,18 @@ typedef struct {
     char * buf;
 } StringBuffer;
 
-int error_context(rage_Error (*f)(StringBuffer b)) {
+// FIXME: This really either ought to go all the way to the top of the testing
```

Comment:
Does "all the way to the top of the testing" mean "test the whole stack" or "run first"?

Reply:
Neither, I mean "should be an integral part of the test runner", will update.